apps/test-stress-can/src/main.cpp

```diff
@@ -0,0 +1,38 @@
+#include "mbed.h"
+#include "CANBus.h"
+#include "CANMsg.h"
+
+CANBus can(CAN_RX, CAN_TX, HWBRIDGE::ROVERCONFIG::ROVER_CANBUS_FREQUENCY);
+CANMsg msg;
+
+DigitalOut led(LED1);
+InterruptIn btn(BUTTON_1);
+
+constexpr uint16_t sent = 1000;
+uint16_t received = 0;
```

Comment:
this could be a static inside of the `receiveMessageOutside` function so u don't accidentally use it in two threads at once. try to always limit the scope of ur variables